openapi_python_client/parser/properties/schemas.py

```diff
@@ -105,3 +115,105 @@ def update_schemas_with_data(
 
     schemas = attr.evolve(schemas, classes_by_reference={ref_path: prop, **schemas.classes_by_reference})
     return schemas
+
+
+@attr.s(auto_attribs=True, frozen=True)
+class Parameters:
+    """Structure for containing all defined, shareable, and reusable parameters"""
+
+    classes_by_reference: Dict[_ReferencePath, Parameter] = attr.ib(factory=dict)
+    classes_by_name: Dict[ClassName, Parameter] = attr.ib(factory=dict)
+    errors: List[ParseError] = attr.ib(factory=list)
```

Comment:
I see where these errors are filled in, but not where they are checked / returned to the user.

Reply:
I think you'll want to add it to `openapi_python_client/parser/openapi.py` line 538.